### DISCUSSION TITLE
Custom domains directories

### DISCUSSION THREAD

**Discussion by ethicnology** [simple-login/app#1505] [Feature Requests]
Is there any prevision to add directories support for custom domain's ex: shop+amazon@custom-domain.xyz ?

**Comment by nguyenkims** [comment]
@ethicnology you can use the auto-create rule on custom domain which is actually more powerful than directory.

**Comment by ethicnology** [comment]
@nguyenkims Thanks for the fast answer, but it doesn't seem to fit my needs well (or maybe I misused it)

The alias overview is pretty crowded when you create an email per account, that's why folders would be interesting for mail created for work, shopping …

**Reply by nguyenkims** [reply]
Oh I see, I think you rather mean this feature https://github.com/simple-login/app/discussions/398: it'll allow a better management of aliases.

**Reply by ethicnology** [reply]
Yes it would be appreciated
